org.eclipse.xtext.testlanguages/src-gen/org/eclipse/xtext/testlanguages/xtextgrammar/services/XtextGrammarTestLanguageGrammarAccess.java

```diff
@@ -360,14 +361,16 @@
 		private final Keyword cSemicolonKeyword_5 = (Keyword)cGroup.eContents().get(5);
 		
 		//ParserRule:
-		//	name=ID ('returns' type=TypeRef)? (definesHiddenTokens?='hidden' '(' (hiddenTokens+=[AbstractRule] (','
-		//	hiddenTokens+=[AbstractRule])*)? ')')? ':'
+		//	name=ID ('returns' type=TypeRef)? (definesHiddenTokens?='hidden' '(' (hiddenTokens+=[AbstractRule] (',' hiddenTokens+=
```

Comment:
bodus formatting

Reply:
Added a unit test and it does what I have expected, looks good for me. Maybe it is the line breaks that confuses you? They are already in the origianal rule definition and are preserved.